status.json

```diff
@@ -5157,5 +5157,20 @@
     },
     "id": 4562033294966784,
     "uservoiceid": 15107964
+  },
+  {
+    "name": "Web Share API",
+    "category": "Device",
+    "link": "https://wicg.github.io/web-share/",
+    "summary": "An API for sharing data (text, URLs, images) from the web to an app of the user's choosing.",
+    "standardStatus": "Editor's draft",
+    "spec": "web-share",
+    "ieStatus": {
+      "text": "Not currently planned"
+    },
+    "impl_status_chrome": "Enabled by default",
```

Comment:
Because `id` was provided, this can be removed.

(I'll do this change before merging).